tensorflow/core/tpu/tpu_initializer_helper.cc

```diff
@@ -133,11 +132,10 @@ bool TryAcquireTpuLock() {
         absl::StrCat(getenv("TPU_LOAD_LIBRARY"));
 
     if (load_library_override == "1") {
-      return true;
+      return Status::OK();
     } else if (load_library_override == "0") {
-      return false;
+      return errors::FailedPrecondition("load library override is not set");
```

Comment:
Let's make this: `errors::FailedPrecondition("TPU_LOAD_LIBRARY=0, not loading libtpu")`

That way if users see this error, it's a bit more clear what's happening